src/unittest/srpe_filter.cpp

```diff
@@ -0,0 +1,349 @@
+/**
```

Comment:
Add a \file to make this a file doc comment.

```
/** \file
 * This documents the file
 */
```

Otherwise you end up documenting the first thing in the file (the vg namespace).